### PR TITLE
Update to OpenTracing 0.12.0

### PR DESCRIPTION
**Pull request by Falco20019**
This currently targets OpenTracing v0.12.0-rc1. I'll update this PR as soon as the final version is released.

### CODE REVIEW

**Comment by Falco20019** [comment]
@cwe1ss Hope you have time for the review. This updates the reference in the `csproj` and also removes the extension methods that were used as intermediate solution but are now part of `OpenTracing 0.12.0` and not needed anymore.

**Review by cwe1ss — APPROVED** [review]
Looks good!

File: src/OpenTracing.Contrib.Grpc/OpenTracing.Contrib.Grpc.csproj [hunk 1]
```diff
@@ -9,8 +9,12 @@
...
   <ItemGroup>
     <!-- Main dependencies -->
-    <PackageReference Include="Grpc" Version="1.11.0" />
-    <PackageReference Include="OpenTracing" Version="0.11.0" />
+    <PackageReference Include="Grpc" Version="1.12.0" />
+    <PackageReference Include="OpenTracing" Version="0.12.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Update="SourceLink.Create.GitHub" Version="2.8.1" />
```
Comment (cwe1ss) [inline src/OpenTracing.Contrib.Grpc/OpenTracing.Contrib.Grpc.csproj]: this shouldn't be necessary as this is done via https://github.com/opentracing-contrib/csharp-grpc/blob/master/src/Directory.Build.props

**Review by cwe1ss — CHANGES_REQUESTED** [review]
see previous comment. feel free to merge once that's resolved!

**Comment by Falco20019** [comment]
Good catch! Visual Studio's NuGet integration is not really good with `props` files...

**Pull request merged**